runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsConfig.java

```diff
@@ -32,32 +32,58 @@
 public interface FpsConfig extends Config
 {
 	@ConfigItem(
-		keyName = "limitMode",
-		name = "Limit Mode",
-		description = "Stay at or under the target frames per second even when in this mode",
+		keyName = "limitFps",
+		name = "Limit Global FPS",
+		description = "Global FPS limit in effect regardless of<br>" +
+			"whether window is in focus or not",
 		position = 1
 	)
-	default FpsLimitMode limitMode()
+	default boolean enableFps()
 	{
-		return FpsLimitMode.NEVER;
+		return true;
```

Comment:
This should be disabled by default to follow previous behaviour

Reply:
It's interesting to me that the default behavior before is "never" by default since users are enabling the plugin to get an FPS limit. It is counterintuitive to need to enable it twice but I will make the change.

Reply:
Well the plugin is also used for drawing FPS what is atm its only feature enabled by default

Reply:
Ah good point, makes sense.